### PR TITLE
Add ability to add dashes to ISBNs.

### PR DESCRIPTION
**Pull request by ataber**

### CODE REVIEW

File: lib/lisbn/lisbn.rb [hunk 1]
```diff
@@ -16,6 +16,16 @@ def valid?
     end
   end
 
+  def isbn_with_dash
+    if valid_isbn_13? && parts
```
Comment (ataber) [inline lib/lisbn/lisbn.rb]: This ignores the tricky case of ISBN10's, maybe this is acceptable for now?

**Comment by ragalie** [comment]
If the ISBN is 13-digits but we don't have `parts`, maybe we can do `#{isbn[0..2]}-#{isbn[3..-1]}`?

**Comment by ataber** [comment]
What do you think about having a dash between the body of the ISBN and the checkdigit? That would work both for ISBN10 and ISBN13.

**Comment by ragalie** [comment]
Sounds good to me!

**Comment by ragalie** [comment]
Can you write a test for it? Otherwise looks good to me!

**Comment by ataber** [comment]
Can I get a thumb in the up direction?

**Comment by ragalie** [comment]
I think I have to merge this for you since it's my repo! :)

**Comment by ataber** [comment]
Thanks! Would you mind please bumping the version as well?

**Pull request merged**